src/test/java/com/adelinasportfolio/emailservice/util/TestConstants.java

```diff
@@ -0,0 +1,5 @@
+package com.adelinasportfolio.emailservice.util;
+
+public class TestConstants {
+    public static final String EMAIL_SENT_SUCCESSFULLY =  "Email sent successfully";
```

Comment:
(acelasi comentariu cu ce am zis in main)

Reply:
Da, o sa crească în timp. Prefer sa am totul organizat de la început.